src/components/FormItem/style.scss

```diff
@@ -0,0 +1,4 @@
+.click:hover {
```

Comment:
nothing wrong with your code, but i think we should all look into css-in-js as a better long-term styling solution
refer to https://emotion.sh/docs/introduction